### PR TITLE
Kriging examples

### PR DESCRIPTION
**Pull request by mbaudin47**
Improved the kriging examples and added new examples 
* kriging the cantilever beam
* kriging a function in 1 dimension and plot the confidence interval
* simulate new trajectories from a kriging
* sequentially add points to a design of experiments based on kriging
* compare several covariance models in terms of regularity of the trajectories of the gaussian processes
* configure the trend and see the difference in terms of number of coefficients to estimate

### CODE REVIEW

**Comment by mbaudin47** [comment]
There is a failing notebook:
```
631/715 Test #1123: pyinstallcheck_notebook_gaussian_processes_comparison ..................***Failed    1.23 sec
Traceback (most recent call last):
  File "/home/devel/project/python/test/t_notebook.py", line 44, in <module>
    exec(source.encode())
  File "<string>", line 197, in <module>
AttributeError: module 'openturns' has no attribute 'viewer'
ipynb_path= /home/devel/project/python/test/../doc/examples
-- gaussian_processes_comparison.ipynb
-- gaussian_processes_comparison.ipynb ***Failed
```

File: python/doc/examples/meta_modeling/kriging.ipynb [hunk 1]
```diff
@@ -11,978 +11,151 @@
...
    "source": [
-    "In this basic example we are going to create a global approximation of a model response using a kriging model.\n",
+    "In this basic example we are going to create an approximation of a model response using a kriging model. We consider a bidimensional function with gaussian inputs. Then we create a kriging metamodel with a constant basis and a `SquaredExponential` covariance. \n",
     "\n",
-    "Here\n",
-    "\n",
-    "$$h(x) = cos(x_1 + x_2)$$"
+    "We consider the function \n",
+    "$$\n",
+    "g(X) = cos(X_1 + X_2)\n",
```
[the remark this answers is not in the document]
Comment (mbaudin47) [reply python/doc/examples/meta_modeling/kriging.ipynb]: Ok.

File: python/doc/examples/meta_modeling/kriging_1D.ipynb [hunk 2]
```diff
@@ -0,0 +1,519 @@
...
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Introduction\n",
+    "\n",
+    "We consider the since function :\n",
+    "$$\n",
+    "y = sin(x)\n",
```
[the remark this answers is not in the document]
Comment (mbaudin47) [reply python/doc/examples/meta_modeling/kriging_1D.ipynb]: Ok.

File: python/doc/examples/meta_modeling/kriging_1D.ipynb [hunk 3]
```diff
@@ -0,0 +1,519 @@
...
+    "In this example, we create a kriging metamodel for a function which has scalar real inputs and outputs. We show how to create the learning and the validation samples. We show how to create the kriging metamodel by chosing a trend and a covariance model. Finally, we compute the predicted kriging confidence interval using the conditional variance."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Introduction\n",
+    "\n",
+    "We consider the since function :\n",
```
[the remark this answers is not in the document]
Comment (mbaudin47) [reply python/doc/examples/meta_modeling/kriging_1D.ipynb]: Ok.

File: python/doc/examples/meta_modeling/kriging_1D.ipynb [hunk 4]
```diff
@@ -0,0 +1,519 @@
...
+    "$$\n",
+    "y = sin(x)\n",
+    "$$\n",
+    "for any $x\\in[0,12]$. \n",
+    "\n",
+    "We want to create a metamodel of this function. This is why we create a sample of $n$ observations of the function :\n",
+    "$$\n",
+    "y_i=sin(x_i)\n",
```
[the remark this answers is not in the document]
Comment (mbaudin47) [reply python/doc/examples/meta_modeling/kriging_1D.ipynb]: Ok.

File: python/doc/examples/meta_modeling/kriging_1D.ipynb [hunk 5]
```diff
@@ -0,0 +1,519 @@
...
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "xmin = 0.\n",
+    "xmax = 12.\n",
+    "n_test = 100\n",
+    "step = (xmax-xmin)/(n_test-1)\n",
+    "myRegularGrid = ot.RegularGrid(xmin, step, n_test)\n",
+    "x_test_coord = myRegularGrid.getValues()\n",
+    "x_test = ot.Sample(x_test_coord,1)\n",
```
[the remark this answers is not in the document]
Comment (mbaudin47) [reply python/doc/examples/meta_modeling/kriging_1D.ipynb]: That's better indeed. Fixed.

File: python/doc/examples/meta_modeling/kriging_1D.ipynb [hunk 6]
```diff
@@ -0,0 +1,519 @@
...
...nFJZvXo1H374Ya5aT4yhenU14TH2M59H+fLl+f3339OTrICAgAxJVk6qVavGypUrSUpKSp9ZePTo0ecL5P85Ojri4uLC0aNHad68OQBpaWmEh4fj4eHxQvcuSCTBMhOpqTBsGNSrp84czI3Dhw9z+/ZtunfvnrfBPScbG3Wg78svq1O/t2wBCwNMu/ivNSuFyCuKorB7925atmyZ7TT6c+fgr79g2jQjB2dgVauqLUS//QZr17akZMmSbNiwwWQSrMKFn701SUvu7u6EhobSqlUrPD09CQgIyNV1ffr04fPPP+fdd9/l008/5caNG8yePRvghb5YjxgxgmnTplGlShWqV6/OTz/9xMOHD03yy7qpklmEZuLnn+H4cZg/P/eDYtevX4+LiwtNmjTJ2+BeQPny6kD37dvVivSGMmLECL777jvD3VCIXDh69CheXl7s378/23M2bFDXHjSX4qI56dNH/duNj7ekS5cubNy4UbqZXkDZsmUJDQ3l3r17eHp6EhcX95/XODg4sHXrVo4fP46Hhweff/45EydOBMgwLutZjRs3jt69ezNgwAAaN25MkSJF8PT0fKF7FjQ6Rf4aciUuLg5HR0diY2NxyM3UPQOKiFCbnvv1UxOt3Bo5ciTW1tY5VgQ2FZ9/DtOnQ1AQtGz54vfr27cvp06d4uTJky9+MyFy6ZNPPmHZsmVERERkW/T25ZehWjW1yKi5u31bHa7w669QqtQOOnbsyKlTp6id07pdBvb48WPOnj1LjRo1KGwOCzoawapVqxg0aBCxsbEUKlTIIPfU6/XUqFEDHx8fpmZTH+jp7+LatWtcvHgRLy+v9C5FLT9DtSJdhGZg9GgoVAi++ebZrvvxxx/zJqA8MHmyOui9Vy+1pc7V9cXu161bN1avXs3FixepWrWqYYIUIgeKorBu3TreeuutbJOrK1fU/78/+8zIweWRMmWgeXO1m3DLljasWbOGihUrah1WgbN8+XIqVapEmTJlOHHiBOPGjcPHx+eFkqvr16+ze/duWrRoQVJSEnPnzuXq1av06dPHgJHnb9JFaOICAtRxSnPmqGuW5dZff/3FkydP8i4wA7OyUt+kn85OSk19sft5eXlRuHBhNmzYYJgAhfgPf/75J9evX6dHjx7ZnrNhg7pY8jMsI2fyevdWW55jYmzp2bNnns16E9mLioqiX79+1KhR
```
[the remark this answers is not in the document]
Comment (mbaudin47) [reply python/doc/examples/meta_modeling/kriging_1D.ipynb]: Ok.

**Comment by mbaudin47** [comment]
@sofianehaddad Please review this PR: it is linked to the PR #1195 that you pushed.

**Comment by mbaudin47** [comment]
> I stop the review waiting for the stabilization of the PR

I am done on this PR: you can review when you want to.

**Comment by jschueller** [comment]
please rebase

**Pull request merged**